src/components/Form.js

```diff
@@ -1,21 +1,48 @@
 import React from 'react';
 import './Form.css';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
 
-const Form = () => (
-  <div>
-    <h2>Add a new book</h2>
-    <form>
-      <input type="text" placeholder="Book title" />
-      <input type="text" placeholder="Author" />
-      <select id="categories-dropdown" name="categories" defaultValue="Categories">
-        <option value="categories" disabled>Categories</option>
-        <option value="action">Action</option>
-        <option value="science fiction">Science Fiction</option>
-        <option value="fantasy">Fantasy</option>
-      </select>
-      <button type="button">ADD BOOK</button>
-    </form>
-  </div>
-);
+const Form = (props) => {
+  const { addBook } = props;
+  return (
+    <div>
+      <h2>Add a new book</h2>
+      <form>
+        <input type="text" placeholder="Book title" id="book-title" />
+        <input type="text" placeholder="Author" id="book-author" />
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/160574711-3bbd5a39-c9fe-4a45-b19e-6a8c2a96f514.png)

Therefore:

- [ ] Please, make these inputs use a React local state in this component.  They should have their inputs that update the local state every time they update (onChange).



## Explanation

We need to make the inputs update the local state (not Redux) every time they update (every keypress from the user).  This is a common React practice with forms.

## Hints

You can import `useState` and use something like `setTitle` and `setAuthor` for the new `onChange` properties on each `<input>`.

Like this:

### Set the state:

```JAVASCRIPT
import React, { useState } from 'react';

const [title, setTitle] = useState('');
const [author, setAuthor] = useState('');
```

### On your inputs:

```JAVASCRIPT
value={title}
onChange={(event) => setTitle(event.target.value)}

value={author}
onChange={(event) => setAuthor(event.target.value)}
```

_Note: This is just an example intended to help.  Your own implementation may vary._

That way, we will meet this particular requirement, and your inputs will update your local state on each keystroke! 💪

Reply:
@Meltrust thanks for the detailed explanation, I just added it!